Expensify/App
# Workspace Admin Translation fix, few es copy changes

@thienlnam 

### Details
Added translation fix for role `Admin` in workspace peoples page. Updated a few incorrect translations in en copies in es file.

### Fixed Issues
$ #4667 

### Tests & QA Steps
1. Go to settings -> Preferreces -> set preferred language as `Espanol`
2. Now go back to the Settings page & open the workspace
3. Tap Personas to see available members in the workspace.

Fix:
4. Role `Admin` should be translated.

Additionally, updated untranslated copy in es, verify that alone.
Copy is for `Beneficial Owners Step` page

### Tested On


### Screenshots


#### Web
<img width="1012" alt="Screenshot 2021-08-31 at 10 31 22 PM" src="https://user-images.githubusercontent.com/85645967/131552377-29726be1-7710-4a32-bcdc-69f36b11f827.png">

#### Mobile Web
![Simulator Screen Shot - iPhone 11 - 2021-08-31 at 22 36 19](https://user-images.githubusercontent.com/85645967/131552375-35e892a6-8501-4d64-b441-f195933f4f2c.png)

#### Desktop
<img width="1190" alt="Screenshot 2021-08-31 at 10 39 34 PM" src="https://user-images.githubusercontent.com/85645967/131552363-8f3949c8-3cd6-442b-b358-ad6fbd678565.png">

#### iOS

![Simulator Screen Shot - iPhone 12 - 2021-08-31 at 23 06 23](https://user-images.githubusercontent.com/85645967/131552347-168a257a-2c67-48ff-8723-43eff517eb68.png)

#### Android
![Screenshot_1630431095](https://user-images.githubusercontent.com/85645967/131552357-948524ff-1445-41c5-94f4-d4547b0b5931.png)
